### PR TITLE
Add inline panel

### PR DESCRIPTION
**Pull request by MahdiMds-Sholex**

### CODE REVIEW

File: LilSholex/groupguard/views.py [hunk 1]
```diff
@@ -1046,7 +1076,7 @@ def webhook(request):
                         raise classes.Violation(group.translate('poll_lock'))
                     elif group.database.game_lock and 'game' in message:
                         raise classes.Violation(group.translate('game_lock'))
-                    elif group.database.inline_keyboard_lock and 'reply_markup' in message:
+                    elif group.database.inline_keyboard_lockinline_keyboard_lock and 'reply_markup' in message:
```
Comment (AntiWanted) [inline LilSholex/groupguard/views.py:1079]: Fix this vairable name !

File: LilSholex/groupguard/views.py [hunk 2]
```diff
@@ -32,6 +34,23 @@ def webhook(request):
             group.delete_message(message_id)
             group.restrict_chat_member(user_id, group.get_chat()['permissions'])
             validation.save()
+        if search('^change_(.*)_lock\S(-.*)', data):
```
Comment (AntiWanted) [inline LilSholex/groupguard/views.py:37]: Check the admin permissions before applying lock changes .

File: LilSholex/groupguard/views.py [hunk 3]
```diff
@@ -39,17 +39,18 @@ def webhook(request):
...
-            keyboard = keyboards.inlinePanel(chat_id, update['callback_query']['from']['id'])
-            if user.database.lang == 'fa':
-                keyboard = keyboard['fa']
-                answer = 'انجام شد !'
-            else:
-                keyboard = keyboard['en']
-                answer = 'Done !'
-            user.edit_message_keyboard(update['callback_query']['from']['id'], message_id, keyboard)
+            user_perms = group.get_chat_member(user.database.chat_id)
+            if user_perms['status'] in ('administrator', 'creator'):
```
Comment (AntiWanted) [inline LilSholex/groupguard/views.py:43]: Check for can_restrict_members permission .

File: LilSholex/groupguard/views.py [hunk 4]
```diff
@@ -40,7 +40,7 @@ def webhook(request):
             user = classes.User(update['callback_query']['from']['id'])
             group = classes.Group(user.database, chat_id)
             user_perms = group.get_chat_member(user.database.chat_id)
-            if user_perms['status'] in ('administrator', 'creator'):
+            if user_perms['status'] in ('administrator', 'creator') or user_perms['can_restrict_members']:
```
Comment (AntiWanted) [inline LilSholex/groupguard/views.py:43]: Change "or" to "and"

Comment (MahdiMds-Sholex) [reply LilSholex/groupguard/views.py]: Changed

**Pull request merged**